gratipay/billing/exchanges.py

```diff
@@ -74,6 +74,7 @@ def create_card_hold(db, participant, amount):
 
     hold = None
     error = ""
+    ref = ""
```

Comment:
Why do we need to set this here? For when the transaction fails and we don't have a ref? Shouldn't we just stick with  null in that case?